packages/edit-post/src/components/layout/style.scss

```diff
@@ -1,21 +1,6 @@
-.edit-post-layout,
-.edit-post-layout__content {
-	height: 100%;
```

Comment:
This one _could_ be the cause of https://github.com/WordPress/gutenberg/pull/18044#issuecomment-557447994.

Reply:
This CSS appears to fix the issue:

```
.edit-post-visual-editor {
	height: 100%;
}
```

But this div also happens to be `editor-styles-wrapper`, and I'm not sure any heights should be applied there?

Reply:
This seems like a good fix to me, It was like that before so I don't see a reason for it to be a problem.

Reply:
Want me to PR?

Reply:
PR'ed: https://github.com/WordPress/gutenberg/pull/18687

Reply:
Thank you